packages/experiments/src/components/fluent/theme/FluentColors.ts

```diff
@@ -26,215 +38,38 @@ export const GrayColors: any = {
-  blue110: '#0027b4',
-  blue100: '#0037da',
-  blue90: '#203dbd',
-  blue80: '#0046ff',
-  blue70: '#2849ec',
-  blue60: '#4f6bed',
-  blue50: '#3b78ff',
-  blue40: '#7c96f9',
-  blue30: '#b5b5e2',
-  blue20: '#bebee5',
-  blue10: '#c3c3f4',
-  blueMagenta250: '#401b6c',
-  blueMagenta240: '#4e257f',
-  blueMagenta230: '#373277',
-  blueMagenta220: '#49397a',
-  blueMagenta210: '#32318c',
-  blueMagenta200: '#5c2e91',
-  blueMagenta190: '#49409a',
-  blueMagenta180: '#5e4a9d',
-  blueMagenta170: '#413eb3',
-  blueMagenta160: '#744da9',
-  blueMagenta150: '#5a4ebc',
-  blueMagenta140: '#735bc1',
-  blueMagenta130: '#8764b8',
-  blueMagenta120: '#4f4bd9',
-  blueMagenta110: '#6b69d6',
-  blueMagenta100: '#7160e8',
-  blueMagenta90: '#886ce4',
-  blueMagenta80: '#8378de',
-  blueMagenta70: '#8e8cd8',
-  blueMagenta60: '#a992d4',
-  blueMagenta50: '#9c89e9',
-  blueMagenta40: '#9a95de',
-  blueMagenta30: '#a68dff',
-  blueMagenta20: '#b4a0ff',
-  blueMagenta10: '#cfc4f5',
-  magenta160: '#5c005c',
-  magenta150: '#460f54',
-  magenta140: '#5c126b',
-  magenta130: '#800074',
-  magenta120: '#721481',
-  magenta110: '#9a0089',
-  magenta100: '#881798',
-  magenta90: '#b4009e',
-  magenta80: '#c239b3',
-  magenta70: '#b146c2',
-  magenta60: '#cc76ca',
-  magenta50: '#c182d1',
-  magenta40: '#e476c1',
-  magenta30: '#de94e0',
-  magenta20: '#e8a3de',
-  magenta10: '#dea2ed',
-  magentaPink70: '#6b0036',
-  magentaPink60: '#77004d',
-  magentaPink50: '#9b0062',
-  magentaPink40: '#bf0077',
-  magentaPink30: '#e3008c',
-  magentaPink20: '#e43ba6',
-  magentaPink10: '#f965bf',
-  pink50: '#970044',
-  pink40: '#c30052',
-  pink30: '#ea005e',
-  pink20: '#ee3f86',
-  pink10: '#edbed3'
-};
-
-export const CommunicationColors: any = {
-  communication90: '#004578',
-  communication80: '#005a9e',
-  communication70: '#106ebe',
-  communication60: '#0078d7',
-  communication50: '#2b88d8',
-  communication40: '#71afe5',
-  communication30: '#c7e0f4',
-  communication20: '#deecf9',
-  communication10: '#eff6fc'
+export const SharedColors: any = {
+  sharedPinkRed10: '#750b1c',
+  sharedRed30: '#a4262c',
+  sharedRed10: '#d13438',
+  sharedRedOrange10: '#da3b01',
+  sharedOrange30: '#8e562e',
+  sharedOrange20: '#ca5010',
+  sharedOrange10: '#ffaa44',
+  sharedYellow10: '#fce100',
+  sharedOrangeYellow20: '#986f0b',
+  sharedOrangeYellow10: '#c19c00',
+  sharedYellowGreen10: '#8cbd18',
+  sharedGreen20: '#0b6a0b',
+  sharedGreen10: '#498205',
+  sharedGreenCyan10: '#00ad56',
+  sharedCyan50: '#005b70',
+  sharedCyan40: '#038387',
+  sharedCyan30: '#00b7c3',
+  sharedCyan20: '#69797e',
+  sharedCyan10: '#a0aeb2',
+  sharedCyanBlue20: '#004e8c',
+  sharedCyanBlue10: '#0078d4',
+  sharedBlue10: '#4f6bed',
+  sharedBlueMagenta30: '#5c2e91',
+  sharedBlueMagenta20: '#8764b8',
+  sharedBlueMagenta10: '#8378de',
+  sharedMagenta20: '#881798',
+  sharedMagenta10: '#c239b3',
+  sharedMagentaPink20: '#9b0062',
+  sharedMagentaPink10: '#e3008c',
+  sharedGray20: '#393939',
```

Comment:
Per @phkuo's comment, should these also be `neutral`?

Reply:
I've been using 'neutral' to mean 'colors that are basically grayscale, including black and white' and 'gray' to mean 'colors that are basically grayscale, excluding black and white'. This is to match how the designers talk about colors.

Reply:
Makes sense to me!

Reply:
Hey were getting rid of what we have now as communicationtint20. We'll need to renumber